src/app/encoder.cpp

```diff
@@ -291,7 +291,7 @@ uint16_t _encodeGlobalCommand(BufBound & buf, EndpointId destination_endpoint, C
     return _encodeCommand(buf, destination_endpoint, cluster_id, command, frame_control);
 }
 
-uint16_t encodeReadAttributesCommand(uint8_t * buffer, uint16_t buf_length, uint8_t destination_endpoint, uint16_t cluster_id,
+uint16_t encodeReadAttributesCommand(uint8_t * buffer, uint16_t buf_length, EndpointId destination_endpoint, uint16_t cluster_id,
```

Comment:
Not `uint8_t` but I guess `uint16_t clusterId` should be 'ClusterId` too ?

Reply:
Good catch; added that.